### ISSUE TITLE
[BUG] Azure.Identity 1.4.0 spelling mistake in ClientCertificateCredentialOptions

### ISSUE THREAD

**Issue by stephenjust** [Azure/azure-sdk-for-net#16328]
**Describe the bug**
In the 1.4.0 beta release of Azure.Identity, there is a spelling mistake in the IncludeX5C**Cliam**Header property.

https://github.com/Azure/azure-sdk-for-net/blob/feature/identity/140/sdk/identity/Azure.Identity/src/ClientCertificateCredentialOptions.cs

**Expected behavior**
The property should be named IncludeX5C**Claim**Header

**Actual behavior (include Exception or Stack Trace)**
The property is named IncludeX5C**Cliam**Header

**To Reproduce**
N/A

**Environment:**
 - Name and version of the Library package used: Azure.Identity 1.4.0-beta.1
 - Hosting platform or OS and .NET runtime version (`dotnet --info` output for .NET Core projects): N/A
 - IDE and version : N/A

**Comment by jsquire** [comment]
Thank you for your feedback.  Tagging and routing to the team member best able to assist.

**Comment by schaabs** [comment]
Thanks @stephenjust for brining this to our attention. We are actually planning on renaming this property to `SendCertificateChain` in our 1.3.0 release which is planned to ship in mid November. We had to ship 1.4.0-beta.1 before releasing 1.3.0 to support a partner team and so it doesn't have this rename which has already been merged to master. I'll leave this issue open to ensure the next release of 1.4.0-beta* has the same rename.
